src/orquestra/sdk/_base/cli/_dorq/_login/_login.py

```diff
@@ -31,6 +32,10 @@ def __init__(
         # data sources
         self._config_repo: _repos.ConfigRepo = config_repo
         self._runtime_repo: _repos.RuntimeRepo = runtime_repo
+        self._token_repo: _repos.TokenRepo = token_repo or _repos.TokenRepo(
+            runtime_repo=self._runtime_repo,
+            presenter=self._login_presenter,
```

Comment:
**thought**: repos are "data" layer. Mixing `TokenRepo` with `LoginPresenter` feels like a lack of separation of concerns. Fortunately `presenter` doesn't seem to be used!